src/components/v2/appDetails/k8Resource/nodeType/NodeTree.component.tsx

```diff
@@ -63,23 +65,23 @@ function NodeTreeComponent({
         });
     }
 
-    const makeNodeTree = (treeNodes: iNodes, parents: string[]) => {
+    const makeNodeTree = (treeNodes: iNodes, parents: string[], isDevtronApp) => {
         return treeNodes.map((treeNode: iNode, index: number) => {
             return (
                 <div key={index + treeNode.name}>
                     <div
                         className={`flex left cursor fw-6 cn-9 fs-14 `}
                         onClick={(e) => handleClickOnNodes(treeNode.name.toLowerCase(), parents, e)}
                     >
-                        {treeNode.childNodes?.length > 0 ? (
+                        {treeNode.childNodes?.length > 0 && !(isDevtronApp && treeNode.name === 'Pod') ? (
```

Comment:
Can avoid hardcoding, instead of 'Pod' can use this data from `Nodes.Pod`

Reply:
done